docs/app/(docs)/commands/scheduleandexpire/page.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: "Command: `` - SudoBot"
```

Comment:
Missing title! 

```dif
- title: "Command: `` - SudoBot"
+ title: "Command: `scheduleandexpire` - SudoBot"
```

Reply:
yeah, there will be a mistake or 2 here and there when im not paying attention, otherwise there thats solved now